### DISCUSSION TITLE
Fix ExplorerPatcher not applying settings on taskbar and changing position will keep saying "Bottom"

### DISCUSSION THREAD

**Discussion by aldeniangman** [valinet/ExplorerPatcher#1893] [Ideas]
fix it
don't say why I am running an insider build
don't close this
don't say ep is not supported on insider build

**Comment by aldeniangman** [comment]
So new update, but nothing fixied!

**Comment by pyrates999** [comment]
Please close this.  Rants do not help anyone.
